docs/sql-data-sources-json.md

```diff
@@ -94,3 +94,171 @@ SELECT * FROM jsonTable
 </div>
 
 </div>
+
+## Data Source Option
+
+Data source options of JSON can be set via:
+* the `.option`/`.options` methods of `DataFrameReader` or `DataFrameWriter`
```

Comment:
Shall we combine this two lines?

Reply:
Thanks for the review, @gengliangwang .

Do you mean combine the 101 and 102 ??

such as

```
the `.option`/`.options` methods of `DataFrameReader` or `DataFrameWriter` or `DataStreamReader` or `DataStreamWriter`
```

???

Reply:
yes, or we can itemize them:
the `.option`/`.options` methods of 
* DataFrameReader
* DataFrameWriter
* ...